internal/cloudentity/client.go

```diff
@@ -0,0 +1,135 @@
+package cloudentity
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+var (
+	HeaderAuthorizerURL = "X-Kusk-Authorizer-URL"
+	HeaderAPIGroup      = "X-Kusk-API-Group"
+)
+
+type Client struct {
+	hc  http.Client
+	url string
+}
+
+func New(url string) *Client {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
`InsecureSkipVerify: true`: mmm. That's quite worrying.

I think a comment here would have been ideal.